flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/descriptors/SchemaValidator.java

```diff
@@ -118,6 +128,33 @@ else if (proctimeFound) {
 				properties.validateExclusion(proctime);
 			}
 		}
+
+		validateWatermark(properties);
+	}
+
+	/**
+	 * Validate watermarks if exists.
+	 */
+	private void validateWatermark(DescriptorProperties properties) {
```

Comment:
I just remembered that  we don't need to validate watermark in connector side. The watermark properteis and computed column properties will be validated and parsed by planner. So we can just remove them from the validation. The only thing that connectors need to do is adding them to the supported list.